src/libutil/json-utils.hh

```diff
@@ -10,6 +10,19 @@ const nlohmann::json * get(const nlohmann::json & map, const std::string & key);
 
 nlohmann::json * get(nlohmann::json & map, const std::string & key);
 
+const nlohmann::json & valueAt(
+    const nlohmann::json & map,
+    const std::string & key);
+
+const nlohmann::json & ensureType(
+    const nlohmann::json & value,
+    nlohmann::json::value_type expectedType);
```

Comment:
Please add documentation to these for the api docs.

Reply:
Done!